examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
Yes, a projection operator linearly mixes channels. Good catch.

If we want both notch and proj, we should notch *all* channels used in projection. The way we were doing it before was wrong -- notching 4 channels then applying proj *should* make the effect of the notch almost disappear. So your new code snippet above is the right thing to do.